src/index.js

```diff
@@ -26,18 +25,18 @@ AddBtn.addEventListener('click', () => {
   }
 });
 
-window.clearAll = () => {
-  tasks = tasks.filter((elem) => elem.completed === false);
+clearAll.addEventListener('click', () => {
+  const tasksArr = tasks.filter((elem) => elem.completed === false);
   // eslint-disable-next-line no-plusplus
-  for (let i = 1; i <= tasks.length; i++) {
```

Comment:
- [ ] As mentioned by the previous reviewer, Kindly make sure you fix all linters errors instead of disabling them.